api/worker-start.ts

```diff
@@ -0,0 +1,4 @@
+const JobQueue = require("./dist/services/JobQueue").default; // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
Looks like you've added worker-start.ts but not deleted worker-start.js. Is this intentional?